README.md

```diff
@@ -117,6 +117,7 @@ Each Lemmy server can set its own moderation policy; appointing site-wide admins
 
 - [lemmy-ui - The official web app for lemmy](https://github.com/LemmyNet/lemmy-ui)
 - [Lemmur - A mobile client for Lemmy (Android, Linux, Windows)](https://github.com/LemmurOrg/lemmur)
+- [Jerboa - A native Android app made by Lemmy's developers](https://github.com/dessalines/jerboa)
```

Comment:
Not sure if we need to emphasize that its made by Lemmy devs (actually just one dev). Imo its enough to state that its a native Android app, as opposed to Lemmur.

Reply:
I think its worth putting, its the tagline I've been using for it in general.